src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsList.jsx

```diff
@@ -11,7 +11,7 @@ class ViewDependentsList extends Component {
       mainContent = (
         <LoadingIndicator message="Loading your dependents" setFocus />
       );
-    } else if (this.props.dependents) {
+    } else if (this.props.dependents && this.props.dependents.length > 0) {
```

Comment:
I think optional chaining could be used here like so:

```js
     } else if (this?.props?.dependents.length > 0) { 
```

Here is a [link to some more info](https://dsva.slack.com/archives/CBU0KDSB1/p1567182149038900)